README.md

```diff
@@ -106,44 +91,77 @@ Starting local OpenShift cluster...
 
    To login as administrator:
        oc login -u system:admin
+```
+Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`.
 
-# Adding 'oc' to the PATH (Note, concrete version might be different)
+ **Note:** By default Minishift uses the driver most relevant to the host OS.
+ To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use: `$ minishift start --vm-driver=virtualbox`.
+
+4. Add `oc` binary to the
+_PATH_ (Note, concrete version might be different):
+ ```sh
```

Comment:
@hferentschik, is this note alright? Note: The following command could vary based on your host OS and the version of the OC binary.